<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUinfo_simple_Form</class>
 <widget class="QWidget" name="CUinfo_simple_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>192</width>
    <height>160</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="systemTypeTitle_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>System Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="systemType_label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>45</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>116</green>
         <blue>108</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="romIDtitle_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>ROM-ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="romID_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>75</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>150</red>
         <green>0</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>150</red>
         <green>0</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>116</green>
         <blue>108</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nrofmbsswstitle_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Measuring Blocks:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nrofdatambstitle_label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>125</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nrofswitchestitle_label">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>125</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Switches:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nrofdatambs_label">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>125</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>116</green>
         <blue>108</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nrofswitches_label">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>125</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>116</green>
         <blue>108</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
